tests/dflowfm/test_mdu.py

```diff
@@ -388,3 +392,115 @@ def test_loading_fmmodel_model_with_both_ini_and_xyn_obsfiles(self):
             assert actual_point.x == expected_point.x
             assert actual_point.y == expected_point.y
             assert actual_point.name == expected_point.name
+
+    def test_mdu_unknown_keywords_loading_gives_message_for_missing_keyword(
+        self, tmp_path, capsys
+    ):
+        tmp_mdu = """
+        [General]
+        fileVersion           = 1.09          
+        fileType              = modelDef      
+        program               = D-Flow FM     
+        version               = 1.2.100.66357 
+        autoStart             = 0             
+        pathsRelativeToParent = 0             
+        unknownkey            = something
+        """
+
+        tmp_mdu_path = tmp_path / "tmp.mdu"
+        tmp_mdu_path.write_text(tmp_mdu)
+
+        with patch(
+            "hydrolib.core.dflowfm.ini.models.INIBasedModel.Config"
```

Comment:
Why do we need this mock?

Reply:
I will remove/update the tests which test with extra, as you also stated in the comment below.